### ISSUE TITLE
Ignore documentation and readmes for Heroku CI

### ISSUE THREAD

**Issue by FireLemons** [rubyforgood/casa#2857]
**Description**
List unnecessary documentation files in a slugignore file to prevent the deploy process when only documentation is updated
see https://devcenter.heroku.com/articles/slug-compiler#ignoring-files-with-slugignore

### Questions? Join Slack!

We highly recommend that you join us in slack https://rubyforgood.herokuapp.com/ #casa channel to ask questions quickly and hear about office hours (currently Wednesday 6-8pm Pacific), stakeholder news, and upcoming new issues.

**Comment by FireLemons** [comment]
I don't think it worked
![image](https://user-images.githubusercontent.com/8918762/139496346-ac17e4a2-2c11-45cd-9aae-6880958ad26b.png)
![image](https://user-images.githubusercontent.com/8918762/139496370-9d811326-3d0a-4016-bac4-4e1159f1789f.png)

**Comment by FireLemons** [comment]
![image](https://user-images.githubusercontent.com/8918762/139496421-956e4040-29db-47f7-8c7a-de0f52776853.png)
What does "it" mean here?

**Comment by FireLemons** [comment]
Maybe the file has to be added to heroku once before it starts ignoring files. I'm keeping this issue open until I know why it's still building.

**Comment by choznerol** [comment]
> ![image](https://user-images.githubusercontent.com/8918762/139496421-956e4040-29db-47f7-8c7a-de0f52776853.png) What does "it" mean here?

My understanding is:
- Add `/spec` in `.glugignore` if not using Heroku CI (like [the example `.slugignore`](https://devcenter.heroku.com/articles/slug-compiler#ignoring-files-with-slugignore))
- Do not add `/spec` in `.glugignore` if using Heroku CI

So "it" means the test suite.

**Comment by choznerol** [comment]
I realized [the docs](https://devcenter.heroku.com/articles/slug-compiler#ignoring-files-with-slugignore) did not says `.slugignore` would prevent unnecessary deploy.

> The .slugignore file causes files to be removed after you push code to Heroku and before the buildpack runs. This lets you prevent large files from being included in the final slug. Unlike .gitignore, .slugignore does not support negated ! patterns.

My understanding is the `.slugignore` is used to avoid big `slug` size. Heroku will warn when slug size > 300MB and stop you from deploying when slug size > 500MB. The past experience when I had to add `.slugignore` is when the huge js bundle size make the slug size > 500MB and all Heroku deployment were rejected.
